en/developer_guide/custom_content_attribute_display.rst

```diff
@@ -0,0 +1,90 @@
+Customize Content Attribute Display
+===================================
+
+Content attributes type are mixed, it's difficult to render them to user. When we
```

Comment:
You could start the sentence with : `To give the minimum constraint on contents, Open Orchestra allows you to store any type of value in the contentAttribute. Sometimes, there is no easy way to render the value to the backoffice user so we have added a stringValue. The stringValue is a projection of the value in a string`...